### DISCUSSION TITLE
Pgwatch3 UI not working

### DISCUSSION THREAD

**Discussion by ondar** [cybertec-postgresql/pgwatch#523] [Q&A] [answered]
I run pgwatch3 with docker compose, go to localhost:8080, login with admin/admin, URL changes to http://localhost:8080/sources and I get white screen:
![image](https://github.com/user-attachments/assets/a06f0913-4f96-4115-a543-f31b8b0f5da0)

When I log in from another computer on the network, I get Undefined error after pressing login:
![image](https://github.com/user-attachments/assets/cf7e72a9-da69-4d2b-b3b6-bae30fdd3b3e)
Seems UI is requesting response from localhost and not the remote server I run it from.

Backend and Grafana seems to be working just fine. What am I doing wrong?

**Comment by pashagolub** [comment, marked as answer]
Thanks! Fixed in #525

**Reply by ondar** [reply]
Thanks, I will try again tomorrow.

**Reply by ondar** [reply]
Thanks. I was able to build new pgwatch image and yeah, 1st problem was solved. UI seems to be running when run on localhost:8080. But still the same problem when run from different computer. Look at the center of the right side of the screen - UI is asking backend on localhost:8080 to login, not the correct computer name and port.

**Reply by pashagolub** [reply]
@ondar would you please provide steps to reproduce this error. Thanks in advance!
